### PR TITLE
Added Lotmaxx sc60 profile

### PR DESCRIPTION
**Pull request by sm3dp**

### CODE REVIEW

**Comment by konskarm** [comment]
Tests are failing. Please have a look and try to fix the issues mentioned in https://github.com/Ultimaker/Cura/pull/7681/checks

**Comment by Ghostkeeper** [comment]
It's probably complaining about the platform mesh file not existing, because it was converted into 3MF recently.

**Comment by Ghostkeeper** [comment]
Developers, merge this at the same time as https://github.com/Ultimaker/fdm_materials/pull/131.

File: resources/definitions/lotmaxx_sc60.def.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    },
+    "metadata": {
+        "visible": true,
+        "author": "lotmaxx.com",
+        "manufacturer": "Lotmaxx",
+        "platform": "lotmaxx_sc_10_20_platform.3mf",
+        "machine_extruder_trains": {
+            "0": "lotmaxx_sc60_extruder_0",
+            "1": "lotmaxx_sc60_extruder_right"
+        },
```
Comment (konskarm) [inline resources/definitions/lotmaxx_sc60.def.json:68]: The left extruder file is named lotmaxx_sc60_**extruder_0**.def.json while the right is named lotmaxx_sc60_**extruder_right**.def.json. Please make them consistent.

**Pull request merged**